.github/workflows/build-test-docker.yml

```diff
@@ -151,15 +152,164 @@ jobs:
         with:
           name: test-results-x86
           path: |
-            test-results-x86
-            test-results-x86-initial
+            test-results-u22-x86
+            test-results-u22-x86-initial
           overwrite: true
       - name: Cleanup
         if: ${{ always() }}
         run: |
-          rm -rf test-results-x86* /tmp/frr-x86-ubuntu22.tar
-          docker stop frr-ubuntu-cont || true
-          docker rm frr-ubuntu-cont || true
+          rm -rf test-results-u22-x86* /tmp/frr-x86-ubuntu22.tar
+          docker stop frr-ubuntu22-cont || true
+          docker rm frr-ubuntu22-cont || true
+
+  build-u24-x86-docker:
```

Comment:
I think the mass duplication is a bad thing here, it's already pretty hard to maintain this stuff. Can we use some variables maybe to share the code?

Reply:
I know. That is exactly what I had in mind and marked it as [WIP]

Reply:
Not only I refactored Ubuntu 22.04 vs 24.04, I also refactored the `arm` targets. I enabled Ubuntu 24.04 both amd64 and arm64.